microshift_storage/dynamic-provisioning-microshift.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="dynamic-provisioning-microshift"]
+= Dynamic provisioning for MicroShift
```

Comment:
Should all instances of hard-coded "MicroShift" in this PR be {product-title} instead?

(Except for topic map of course)